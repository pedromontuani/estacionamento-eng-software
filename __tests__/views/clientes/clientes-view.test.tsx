import React from 'react';
import {rendererCreate} from '../../utils';

import ClientsView from '../../../src/views/clientes/clientes-view';

describe('Testing clients view', () => {
  test('should match snapshot', async () => {
    const tree = rendererCreate(<ClientsView />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
