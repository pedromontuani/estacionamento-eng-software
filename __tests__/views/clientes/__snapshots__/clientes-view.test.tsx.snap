// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing clients view should match snapshot 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "padding": 24,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "justifyContent": "space-evenly",
      }
    }
  >
    <View
      accessibilityRole="button"
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Array [
            Object {},
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#007AFF",
                "fontSize": 18,
                "margin": 8,
                "textAlign": "center",
              },
            ]
          }
        >
          CHECK-IN
        </Text>
      </View>
    </View>
    <View
      accessibilityRole="button"
      accessible={true}
      collapsable={false}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Array [
            Object {},
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#007AFF",
                "fontSize": 18,
                "margin": 8,
                "textAlign": "center",
              },
            ]
          }
        >
          CHECK-OUT
        </Text>
      </View>
    </View>
  </View>
  <Modal
    hardwareAccelerated={false}
    onRequestClose={[Function]}
    transparent={true}
    visible={false}
  />
  <Modal
    hardwareAccelerated={false}
    onRequestClose={[Function]}
    transparent={true}
    visible={false}
  />
</View>
`;
